action.yml

```diff
@@ -0,0 +1,30 @@
+name: pysa
+description: "Run pysa on the repository and check for vulnerabilities"
+
+inputs:
+  build-type:
+    description: Pysa build to use. Possible values are stable and nightly. Default is stable.
+    required: false
+    default: "stable"
+
+runs:
+  using: "composite"
+  steps:
+    - run: if [ ${{inputs.build-type}} = "nighty" ]; then pip install pyre-check-nightly; else pip install pyre-check; fi
+      shell: bash
+    - run: if [ ! -f .pyre_configuration ]; then printf ".\n" | pyre init; fi
+      shell: bash
+    - run: pyre infer
+      shell: bash
+    - run: pyre infer -i --annotate-from-existing-stubs
+      shell: bash
+    - run: pip install fb-sapp
+      shell: bash
+    - run: mkdir ./.pysa/
+      shell: bash
+    - run: pyre analyze --no-verify --save-results-to ./
+      shell: bash
+    - run: sapp analyze ./taint-output.json
```

Comment:
We also need to run `sapp update warning-codes ./taint-metadata.json` here before running `sapp filter issues`. 

The `update warning-codes` command adds the list of warning codes present in the Pysa run into SAPP db to reference when creating the rules property in the SARIF output